GetDKAN/dkan-tools
# 17 custom commands

Fixes #17 
Merge After #25 

**QA**
- Run 'dktl init'
- Create the class given in the README.md into src/command/CustomCommands.php
- Run 'dktl custom:sample'
- Should see "Hello World!!" printed
